app/viewers/embed/viewer/mirador_viewer.rb

```diff
@@ -34,7 +34,7 @@ def cdl?
       # - ensure it exists in the manifest (if they don't, mirador puts the user into a weird initial state)
       # - rewrite pre-cocina canvas ids to post-cocina canvas ids as appropriate
       #        (to avoid breaking embeds that used to work)
-      # rubocop:disable Metrics/AbcSize
+      # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Is there a reason these were added? It doesn't look like anything changed so it is a little weird it was added